packages/bot-web-ui/src/components/dashboard/tutorial-tab/guide-content.tsx

```diff
@@ -1,41 +1,33 @@
 import { Dialog, Icon, Text } from '@deriv/components';
 import { isMobile } from '@deriv/shared';
 import { localize } from '@deriv/translations';
+import { observer } from '@deriv/stores';
 import classNames from 'classnames';
 import { DBOT_TABS } from 'Constants/bot-contents';
 import React from 'react';
-import { connect } from 'Stores/connect';
-import RootStore from 'Stores/index';
+import { useDBotStore } from 'Stores/useDBotStore';
 import { removeKeyValue } from '../../../utils/settings';
 import { tour_type } from '../joyride-config';
 
 type TGuideContent = {
-    dialog_options: { [key: string]: string };
-    faq_search_value: string;
     guide_list: [];
-    is_dialog_open: boolean;
-    onOkButtonClick: () => void;
-    setActiveTab: (tab_title: number) => void;
-    setHasTourEnded: (param: boolean) => boolean;
-    setOnBoardTourRunState: (param: boolean) => boolean;
-    setTourActive: (param: boolean) => boolean;
-    setTourDialogVisibility: (param: boolean) => boolean;
-    showVideoDialog: (param: { [key: string]: string }) => void;
 };
 
-const GuideContent = ({
-    dialog_options,
-    faq_search_value,
-    guide_list,
-    is_dialog_open,
-    onOkButtonClick,
-    setActiveTab,
-    setHasTourEnded,
-    setOnBoardTourRunState,
-    setTourActive,
-    setTourDialogVisibility,
-    showVideoDialog,
-}: TGuideContent) => {
+const GuideContent = observer(({ guide_list }: TGuideContent) => {
+    const { dashboard } = useDBotStore();
+    const {
+        dialog_options,
+        faq_search_value,
+        is_dialog_open,
+        onCloseDialog: onOkButtonClick,
```

Comment:
Bhai, would it be good if we kept it the same instead of aliasing it? May need to change where it is being used but if the changes are minor can we do it?